swarmlib/pso/main.py

```diff
@@ -36,7 +36,7 @@ def configure_parser(sub_parsers):
         type=str,
         default='michalewicz',
         help='Choose the function that is used for searching the minimum.',
-        choices=['michalewicz', 'ackley'])
+        choices=[*FUNCTIONS])
```

Comment:
Landscapes also includes 3D and 4D functions. I like the idea but won't this line falsely show those not supported functions as choice as well?

Reply:
Your're right, I missed that. I'll find a way to exclude 3D and 4D functions from `FUNCTIONS`. There are only two functions that fall into this category atm (`wolfe()` and `colville()`)

Reply:
Done